config/clock.rb

```diff
@@ -0,0 +1,10 @@
+require "clockwork"
+require_relative "boot"
+require_relative "environment"
+
+module Clockwork
+  every(1.day, "scheduled_messages.send", at: "19:30", tz: "UTC") do
+    puts "Sending scheduled messages"
+    MessageSender.new.delay.run
```

Comment:
I am running it with delayed job so that the jobs all get saved into the DB and re-run if they fail. So still want `delay` but do need to update setup